vendor/bat-native-ads/src/bat/ads/internal/account/redeem_unblinded_token/redeem_unblinded_token.cc

```diff
@@ -139,12 +139,6 @@ void RedeemUnblindedToken::OnFetchPaymentToken(
   if (url_response.status_code == net::HTTP_NOT_FOUND) {
     BLOG(1, "Confirmation not found");
 
-    if (!security::Verify(confirmation)) {
-      BLOG(1, "Failed to verify confirmation");
-      OnFailedToRedeemUnblindedToken(confirmation, /* should_retry */ false);
-      return;
-    }
-
```

Comment:
Removed to support upgrade paths from v1 endpoint to v2 endpoint where confirmations are missing so we recreate the confirmation on the new endpoint after a retry